GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -100,31 +97,17 @@ internal Keys GetShortcutKeys(Commands cmd)
 
         #endregion
 
-        public string GetTabText()
+        public void GetTabText()
```

Comment:
Should this method be renamed? It doesn't seem to do what its name suggests.

Reply:
It gets the text for the diff tab. Is DisplayDiffTab() better?